src/lib/y2partitioner/dialogs/partition_type.rb

```diff
@@ -15,7 +15,7 @@ class TypeChoice < CWM::RadioButtons
         def initialize(controller)
           textdomain "storage"
           @controller = controller
-          @slots = controller.unused_slots
+          @slots = controller.unused_optimal_slots
```

Comment:
Because optimal slots are the base for the two first options (max size and custom size) and they are kind of a subset of `unused_slots` (whatever is possible with optimal slots is also possible with non optimal).

If, by any change, using non-optimized slots would allow more possibilities than the optimized one (very strange but can happen) and we would allow that type to be used, the first two options would become bogus.

Reply:
I would add a comment but I plan to move this logic to the controller and that should make the intention more clear.